### PR TITLE
feat(PE-4210): added schemas and contract calls

### PR DESCRIPTION
**Pull request by atticusofsparta**
ant contract deployed with the compiled source code https://sonar.warp.cc/#/app/contract/OkAq3Rlkd_qURQIiqKqlvkgQo56xa4YzbDO8arpPOGE?network=mainnet#

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,9 +10,11 @@
     "test": "jest tests"
   },
   "dependencies": {
+    "@types/webpack-env": "^1.18.1",
```
Comment (dtfiedler) [inline package.json:13]: why?

File: build.js [hunk 2]
```diff
@@ -1,7 +1,52 @@
...
+  setTickerSchema,
+  setRecordSchema,
+  removeRecordSchema,
+  transferTokensSchema,
+} = require("./schemas");
+
+
+
+// build our validation source code
+console.log(typeof balanceSchema)
```
Comment (dtfiedler) [inline build.js:22]: remove

File: package.json (hunk 1 above)
Comment (atticusofsparta) [reply package.json]: Dunno, works without it so i removed it.

File: build.js (hunk 2 above)
Comment (atticusofsparta) [reply build.js]: removed

**Pull request merged**